external-crates/move/crates/move-compiler/tests/new_rules/constant_naming.exp

```diff
@@ -0,0 +1,32 @@
+warning[Lint W06001]: Constant name should be in all caps and snake case.
```

Comment:
Are these being run? I'm surprised if they are.  
But we will need to make a new directory for the new lints